modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -1734,58 +1687,61 @@ void GDScriptAnalyzer::resolve_match_pattern(GDScriptParser::PatternNode *p_matc
 }
 
 void GDScriptAnalyzer::resolve_parameter(GDScriptParser::ParameterNode *p_parameter) {
-	GDScriptParser::DataType result;
-	result.kind = GDScriptParser::DataType::VARIANT;
+	GDScriptParser::DataType type;
```

Comment:
I actually like that this is named result, and I think that's the pattern elsewhere as well, so I'd personally avoid renaming here.

Reply:
It is called `type` in `resolve_variable` and `resolve_constant`. The other places that this code has to be consistent with and potentially merged with in the future.

Reply:
My bad then :) I think a lot of the `reduce_*` functions I was working with used `result`, that's why I made this comment! But if the `resolve_*` functions are now more standardized, perfect!